data/presets/shop/frozen_food.json

```diff
@@ -1,5 +1,5 @@
 {
-    "icon": "maki-shop",
+    "icon": "far-snoflake",
```

Comment:
```suggestion
    "icon": "fas-snowflake",
```

Reply:
tried fas (solid) prefix in https://github.com/openstreetmap/id-tagging-schema/pull/839 - and failed anyway

Reply:
I was trying to point out that it should be `snowflake` instead of `snoflake`